src/arch/aarch64/ram64.s

```diff
@@ -0,0 +1,32 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright (C) 2022 Akira Moroo */
+
+.section .text.boot, "ax"
+.global ram64_start
+
+ram64_start:
+  /* UEFI "MZ" signature magic instruction */
+  add x13, x18, #0x16   /* code0 */
```

Comment:
Maybe I lost the context here. I think ram64_start is the entry code. Could we use general register without initialization here? what's the meaning of this line?

Reply:
Please read https://github.com/cloud-hypervisor/rust-hypervisor-firmware/pull/205#issuecomment-1327301458.

Reply:
Another question. I think this "MZ" signature should be part of PE file header not the part of text segment. Thus, should these header code be moved out of text.boot segment?